src/main/java/io/cdap/plugin/servicenow/source/ServiceNowSourceConfig.java

```diff
@@ -173,7 +163,7 @@ public SourceQueryMode getQueryMode(FailureCollector collector) {
     }
 
     collector.addFailure("Unsupported query mode value: " + queryMode,
-      String.format("Supported modes are: %s", SourceQueryMode.getSupportedModes()))
+        String.format("Supported modes are: %s", SourceQueryMode.getSupportedModes()))
```

Comment:
remove added whitespaces/indent changes here and elsewhere

Reply:
the extra whitespaces/indent changes have been removed. 9753ec9e11296d5d686020e54a7f2716cf18eb04